package.json

```diff
@@ -9,6 +9,7 @@
     "dotenv": "^6.0.0",
     "express": "^4.16.3",
     "helmet": "^3.13.0",
+    "jshint": "^2.9.6",
```

Comment:
You'll probably want to install this globally, or as a dev dependency at a minimum. The team also decided on eslint as well IIRC.